etc/cmake/options.cmake

```diff
@@ -86,6 +86,7 @@ ot_option(OT_ASSERT OPENTHREAD_CONFIG_ASSERT_ENABLE "assert function OT_ASSERT()
 ot_option(OT_BACKBONE_ROUTER OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE "backbone router functionality")
 ot_option(OT_BACKBONE_ROUTER_DUA_NDPROXYING OPENTHREAD_CONFIG_BACKBONE_ROUTER_DUA_NDPROXYING_ENABLE "BBR DUA ND Proxy")
 ot_option(OT_BACKBONE_ROUTER_MULTICAST_ROUTING OPENTHREAD_CONFIG_BACKBONE_ROUTER_MULTICAST_ROUTING_ENABLE "BBR MR")
+ot_option(OT_BBTC OPENTHREAD_CONFIG_BLE_SECURE_ENABLE "Ble based thread commissioning")
```

Comment:
I believe the feature is now named Thread Commissioning over Authenticated TLS (TCAT). Can we update to be consistent with the latest naming?

Reply:
TCAT is the commonality of (potentially) multiple link technologies using the same protocol semantics. Bluetooth Based Thread Commissioning (BBTC) has TCAT as a dependency. Likewise future 15.4 based direct commissioning methods would also use the common TCAT agent implementing the TLV protocol.
I suggest the feature should be called BBTC to differentiate it from other features implementing TCAT with another link-layer. The specification draft uses both abbreviations.

Reply:
In the draft spec, I see the term "BLE TCAT", but I do not see "BBTC". I would prefer the OpenThread feature names to map closely to the Thread Specification when possible.

Reply:
May be we should read BBTC as Bluetooth Based TCat.

Reply:
I think it can help new developers and integrators when they see the "BLE" term in the option/config name explicitly, and it is not that long. :) (`BLE_TCAT` sounds good to me).

Would also help keeping the CMake option name close to its related `OPENTHREAD_CONFIG_{name}` (easier for folks reading the code to tell they are related).

Reply:
I just realized the term BBTC has been eliminated from the specification and replaced with "TCAT over a BLE". So yes, BLE_TCAT makes perfect sense.